### DISCUSSION TITLE
A log about gRPC alarm

### DISCUSSION THREAD

**Discussion by teamoTNT** [apache/skywalking#10709] [General]
![image](https://user-images.githubusercontent.com/103160892/233562296-2a364683-96af-4ebe-bedf-a0741f92ca13.png)

This log keeps output, whether I have set up gRPC alarm or not.
About source(org.apache.skywalking.oap.server.core.alarm.provider.grpc.GRPCCallback#onGRPCAlarmSettingUpdated):
![image](https://user-images.githubusercontent.com/103160892/233561246-83918939-a989-4830-92b6-a1e44aa05155.png)
Maybe the log can move like this:
![image](https://user-images.githubusercontent.com/103160892/233561525-c50a668e-bacd-4b9e-9061-2e4df0444e5b.png)

**Comment by wu-sheng** [comment]
I think it is better moving this `warning` to L145 after the shutdown as `debug` log only.

**Reply by wu-sheng** [reply]
https://github.com/apache/skywalking/pull/10712

**Reply by teamoTNT** [reply]
Wow, great approach.
